### DISCUSSION TITLE
horizontal scrollbar disappears and never comes back

### DISCUSSION THREAD

**Discussion by Senajs** [mikekazakov/nimble-commander#156] [Q&A]
Hi all,

i have the strange issue that when I work with NC the horizontal scrollbar disappears. Now this is a setting in MacOS (yes I know) but: no matter how much I move the MOUSE it never comes back (the scrollbar). If I move/Touch the touchpad it DOES come back only to disappear again as long as I move the mouse.. 

has anyone an idea why or how ?

KR

**Comment by mikekazakov** [comment]
Answered in #272.
